liberty-maven-plugin/src/it/tests/appsdirectory-apps-configured-variables-it/src/test/java/net/wasdev/wlp/maven/test/app/PluginConfigXmlTest.java

```diff
@@ -0,0 +1,36 @@
+package net.wasdev.wlp.maven.test.app;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathFactory;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+/**
+ * 
+ * Web application test case
+ * 
+ */
+
+public class PluginConfigXmlTest {
```

Comment:
Will fix it. 
Actually the test was not intended to check the existence of the install_apps*.xml. As I indicated in the comment, it was a quick workaround  to check if the variables are resolved correctly then the file wouldn't be generated in this test case.